rules/Symfony73/Rector/Class_/InvokableCommandInputAttributeRector.php

```diff
@@ -147,6 +148,7 @@ public function refactor(Node $node): ?Class_
         }
 
         $executeClassMethod->name = new Identifier('__invoke');
+        $executeClassMethod->flags = Modifiers::PUBLIC;
```

Comment:
use `\Rector\Privatization\NodeManipulator\VisibilityManipulator::makePublic()` instead to avoid invalid replace in case of there is another flag combination.

https://github.com/rectorphp/rector-src/blob/8dfc7aa424967b1f8e509e125706f19c0cf9cbb6/rules/Privatization/NodeManipulator/VisibilityManipulator.php#L92-L95

Reply:
Done!